### PR TITLE
[script bundle] Add action to update script bundle in gh-pages

### PR DESCRIPTION
**Pull request by JamesMBartlett**
Summary: Adds a manually triggered action to update the script bundle stored in github pages.

Type of change: /kind cleanup.

Test Plan: Tested on my fork. Saw that it adds the bundle to gh-pages as expected. Also tested that a rerun of the workflow with no changes won't attempt an empty commit.

### CODE REVIEW

File: .github/workflows/update_script_bundle.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      uses: ./.github/actions/bazelrc
+      with:
+        use_remote_exec: 'true'
+        BB_API_KEY: ${{ secrets.BB_IO_API_KEY }}
+        download_toplevel: 'true'
+    - name: Install Pixie CLI
+      run: |
+        bazel build -c opt //src/pixie_cli:px
+        p="$(bazel cquery -c opt //src/pixie_cli:px --output starlark --starlark:expr 'target.files.to_list()[0].path')"
+        cp "${p}" /usr/bin/px
```
Comment (vihangm) [inline .github/workflows/update_script_bundle.yaml:34]: should we use a released CLI and run this without the bazel setup?

**Review by vihangm — APPROVED** [review]

**Review by vihangm — APPROVED** [review]
